### PR TITLE
fix: 修复应用部署时数据关联错误的Bug

### LINKED ISSUE #177: 应用部署失败，数据错乱
### 运行环境
* `应用版本`:
     * `atomci-frontend`: 1.5.1
     * `atomci`:1.5.1
* `Kubernetes`:1.24

### 截图
6b13639fb4a7fcf0b4c6214ae0cd2e8

### PR DESCRIPTION
**Pull request by sampsonye**
### Which changes (Bug/Feature):

Fixes #177 

### Special notes for reviewers:

```
none
```

### CODE REVIEW

**Review by colynn — APPROVED** [review]
Some doubts？

File: internal/core/pipelinemgr/uitls.go [hunk 1]
```diff
@@ -987,6 +986,7 @@ func (pm *PipelineManager) aggregateAppsParamsForDeploy(publishID, stageID int64
 			ScmApp:          scmApp,
 			RunDeployAppReq: app,
```
Comment (colynn) [inline internal/core/pipelinemgr/uitls.go:987]: RunDeployAppReq 的结构内就包含 project_app_id ,

**Review by colynn — CHANGES_REQUESTED** [review]
应用部署的jenkins job 是 "aomci_" +  projectID  +  stageID  组成, 没有 project_app_id的元素

File: internal/core/pipelinemgr/uitls.go (hunk 1 above)
Comment (sampsonye) [reply internal/core/pipelinemgr/uitls.go]: 确实重复定义了，已移除

**Review by colynn — APPROVED** [review]
LGTM

**Pull request merged**